assets/js/index.js

```diff
@@ -192,7 +199,14 @@ class ScrollListener {
 
   bindScroll() {
     this.checkIfNewTarget();
-    document.addEventListener('scroll', this.checkIfNewTarget.bind(this));
+
+    document.addEventListener('scroll', () => {
+      clearTimeout( this.isScrolling );
+        this.isScrolling = setTimeout(() => {
+          this.checkIfNewTarget()
+        }, 60);
+    });
```

Comment:
Are we just throttling here?

Reply:
Yes. I'm trying to execute the ```checkIfNewTarget``` function only when the scroll stops. This fixes [this](https://cl.ly/c535757e8baf) issue